private/buf/cmd/buf/command/beta/convert/convert.go

```diff
@@ -115,7 +119,7 @@ func run(
 	if err := bufcli.ValidateErrorFormatFlag(flags.ErrorFormat, errorFormatFlagName); err != nil {
 		return err
 	}
-	input, err := bufcli.GetInputValue(container, "", "")
+	input, err := bufcli.GetInputValue(container, flags.InputHashtag, ".")
```

Comment:
```go
"."
```
is this referencing the current context?

Reply:
Yes. Before doing things like `buf beta convert -#format=json` was not supported but now it is so the "." is setting that the current context is the current directory just like with `buf build` (for the input image)